go/install/libnativeinstaller/O_O.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2017 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package libnativeinstaller
+
+import (
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/kardianos/osext"
+	"github.com/keybase/client/go/libkb"
+)
+
+// Log is the logging interface for this package
+type Log interface {
+	Debug(s string, args ...interface{})
+	Info(s string, args ...interface{})
+	Warning(s string, args ...interface{})
+	Errorf(s string, args ...interface{})
+}
+
+// Context is the enviroment for install package.
+type Context interface {
+	GetConfigDir() string
+	GetCacheDir() string
+	GetRuntimeDir() string
+	GetMountDir() (string, error)
+	GetLogDir() string
+	GetRunMode() libkb.RunMode
+	GetServiceInfoPath() string
+	GetKBFSInfoPath() string
+}
+
+// AppBundleForPath returns path to app bundle
+func AppBundleForPath() (string, error) {
```

Comment:
Is there a reason you didn't remove the corresponding function from `go/install/install_darwin.go`?

Reply:
Nope! Good catch.